src/tools/website/resources/grunt-tasks/grunt-copy-website-content.js

```diff
@@ -158,7 +158,7 @@ module.exports = function(grunt) {
       };
 
       this.ensureAbsoluteLinkPaths = function(filepath, text) {
-        return text.replace(/\[(.+)\]\(([^\)]+)\)/gi, function(
+        return text.replace(/\[([^\]]+)\]\(([^\)]+)\)/gi, function(
```

Comment:
@markus2330 The issue actually wasn't that `doc/KEYNAMES.md` was not listed in `structure.json.in` (although I fixed that as well). The issue was that we don't use a proper markdown parser and this Regex only worked, if there was just one link per line.

See also: https://regex101.com/r/OeDKOS/1